saleor/graphql/product/tests/test_product.py

```diff
@@ -9668,6 +9673,65 @@ def test_update_or_create_variant_stocks_empty_stocks_data(variant, warehouses):
     assert stock.quantity == 5
 
 
+@patch("saleor.plugins.webhook.plugin.trigger_webhooks_for_event.delay")
+def test_update_or_create_variant_stocks_empty_stocks_data_with_variant_stocks_webhooks(
+    mocked_webhook_trigger, settings, variant, warehouses, info
+):
+    Stock.objects.create(
+        product_variant=variant,
+        warehouse=warehouses[0],
+    )
+    settings.PLUGINS = ["saleor.plugins.webhook.plugin.WebhookPlugin"]
+
+    info.context.plugins = get_plugins_manager()
+    stocks_data = [
+        {"quantity": 10, "warehouse": "123"},
+    ]
+
+    ProductVariantStocksUpdate.update_or_create_variant_stocks(
+        variant, stocks_data, warehouses, info
+    )
+
+    assert (
```

Comment:
You're testing here partially `PluginsManager` and `WebhookPlugin` functionality. 
It would be better if you would patch directly PluginsManager calls. Something like this:
`@patch("saleor.plugins.manager.PluginsManager.order_updated")`
You can use two patches for each method, then you can use `assert_called_once_with`.  Thanks to this, there is no need to worry about order of method calls (currently you're using call args list)